src/schedule_gantt.py

```diff
@@ -0,0 +1,125 @@
+# -*- coding: utf-8 -*-
+import json
+import matplotlib.pyplot as plt
+import matplotlib.colors as clrs
+import sys
+
+# Very simple plot for a VROOM solution file.
+
+colors_blacklist = [
+    "whitesmoke",
+    "white",
+    "snow",
+    "mistyrose",
+    "seashell",
+    "linen",
+    "bisque",
+    "antiquewhite",
+    "blanchedalmond",
+    "papayawhip",
+    "wheat",
+    "oldlace",
+    "floralwhite",
+    "cornsilk",
+    "lemonchiffon",
+    "aliceblue",
+    "ivory",
+    "beige",
+    "lightyellow",
+    "lightgoldenrodyellow",
+    "honeydew",
+    "mintcream",
+    "azure",
+    "lightcyan",
+    "aliceblue",
+    "ghostwhite",
+    "lavender",
+    "lavenderblush",
+]
+
+
+def plot_schedules(sol_file_name):
+    plot_file_name = sol_file_name[0 : sol_file_name.rfind(".json")] + "_gantt.svg"
+
+    print("Parsing " + sol_file_name)
+    with open(sol_file_name, "r") as sol_file:
+        solution = json.load(sol_file)
+
+    color_list = []
+    for name, hex in clrs.cnames.items():
+        if name not in colors_blacklist:
+            color_list.append(name)
+
+    fig, ax1 = plt.subplots(1, 1)
+    fig.set_figwidth(15)
+    plt.subplots_adjust(left=0.03, right=1, top=1, bottom=0.05, wspace=0.03)
+
+    if "routes" not in solution:
+        return
+
+    t = 0
+    dt = 0
```

Comment:
Initial values are never used so this pre-definition can probably be removed altogether.